packages/client/src/components/CrazyTitle/CrazyTitle.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react'
```

Comment:
Этот импорт не используется, поэтому лучше удалить из файлов

```suggestion
import { FC } from 'react'
```